doc/source/users_guide/index.rst

```diff
@@ -51,69 +51,102 @@ This example shows how you can display a mesh:
         "solid_up:1:830",
         "solid_up:1:830-shadow",
     ]
-    mesh1.display("window-1")
+
+    mesh1 = Mesh(solver=solver_session, show_edges=True, surfaces=mesh_surfaces_list)
+    p1 = GraphicsWindow(grid=(1, 2))
+    p1.add_graphics(mesh1, position=(0, 0))
+
+    mesh2 = Mesh(solver=solver_session, surfaces=mesh_surfaces_list)
+    mesh2.show_edges = False
+
+    p1.add_graphics(mesh2, position=(0, 1))
+    p1.show()
+
+Display plane-surface
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    from ansys.fluent.visualization import Surface
+
+    surf_xy_plane = Surface(solver=solver_session)
+    surf_xy_plane.definition.type = "plane-surface"
+    surf_xy_plane.definition.plane_surface.creation_method = "xy-plane"
+    plane_surface_xy = surf_xy_plane.definition.plane_surface.xy_plane
+    plane_surface_xy.z = -0.0441921
```

Comment:
@prmukherj I have overlooked classes like this one in the refactoring. This awkward construction sequence is inconsistent with what we achieved for the graphical classes.

Reply:
Yes, totally agreed. Will encapsulate it today.